app/models/runner.js

```diff
@@ -137,12 +151,16 @@ async function writeProcessorFile(fileContent) {
     }
 }
 
-function injectPlugins(testFile, jobConfig) {
-    const metricsPluginName = jobConfig.metricsPluginName.toLowerCase();
-    const metricsAdapter = require(`../adapters/${metricsPluginName}Adapter`);
-    let asciiMetricsExportConfig = (Buffer.from(jobConfig.metricsExportConfig, 'base64').toString('ascii'));
-    let parsedMetricsConfig = JSON.parse(asciiMetricsExportConfig);
-    testFile.config.plugins = metricsAdapter.buildMetricsPlugin(parsedMetricsConfig, jobConfig);
+async function getProcessorPathIfExists(jobConfig, test) {
+    let localProcessorPath;
+    if (test['file_id']) {
```

Comment:
maybe add log saying it's deprecated? and put a link to processors docs?

Reply:
done